.changeset/sour-months-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ebay/ebayui-core": patch
```

Comment:
Since this is going into `13.3.0` it should be a `minor` release, right?

Reply:
This change is a patch type change since it's a bugfix not a feature. 
So what will happen is the largest changeset will be picked when it's together with a bunch of other changes. 

So if we have a bunch of patches together with a minor change, the change will be minor because minor is the "largest" type changeset.

Generally I've been doing patch on bugfix changes, minor on features, and major on breaking changes.